Failed to build sentry-native 0.4.16 with no examples

When building just sentry-native lib without examples, it's fails to build:

```
   Creating library sentry.lib and object sentry.exp
sentry_core.c.obj : error LNK2019: unresolved external symbol _sentry_transaction_set_name referenced in function _sentry_set_transaction
sentry_core.c.obj : error LNK2019: unresolved external symbol _sentry__transaction_context_free referenced in function _sentry_transaction_start
sentry_core.c.obj : error LNK2019: unresolved external symbol _sentry__transaction_new referenced in function _sentry_transaction_start
sentry_core.c.obj : error LNK2019: unresolved external symbol _sentry__transaction_incref referenced in function _sentry_set_transaction_object
sentry_core.c.obj : error LNK2019: unresolved external symbol _sentry__transaction_decref referenced in function _sentry_transaction_finish
sentry_scope.c.obj : error LNK2001: unresolved external symbol _sentry__transaction_decref
sentry_core.c.obj : error LNK2019: unresolved external symbol _sentry__span_incref referenced in function _sentry_set_span
sentry_core.c.obj : error LNK2019: unresolved external symbol _sentry__span_decref referenced in function _sentry_set_transaction_object
sentry_scope.c.obj : error LNK2001: unresolved external symbol _sentry__span_decref
sentry_core.c.obj : error LNK2019: unresolved external symbol _sentry__value_span_new referenced in function _sentry_transaction_start_child
sentry_core.c.obj : error LNK2019: unresolved external symbol _sentry__span_new referenced in function _sentry_transaction_start_child
sentry_core.c.obj : error LNK2019: unresolved external symbol _sentry__span_free referenced in function _sentry_span_finish
sentry_core.c.obj : error LNK2019: unresolved external symbol _sentry__value_get_trace_context referenced in function _sentry_transaction_finish
sentry_scope.c.obj : error LNK2001: unresolved external symbol _sentry__value_get_trace_context
sentry.dll : fatal error LNK1120: 11 unresolved externals
```

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -42,6 +42,8 @@ sentry_target_sources_cwd(sentry
 	sentry_uuid.h
 	sentry_value.c
 	sentry_value.h
+	sentry_tracing.c
+	sentry_tracing.h
 	path/sentry_path.c
 	transports/sentry_disk_transport.c
 	transports/sentry_disk_transport.h
@@ -149,12 +151,5 @@ if(SENTRY_INTEGRATION_QT)
 	sentry_target_sources_cwd(sentry
 		integrations/sentry_integration_qt.cpp
 		integrations/sentry_integration_qt.h
-	)
-endif()
-
-if(SENTRY_BUILD_EXAMPLES)
-	sentry_target_sources_cwd(sentry
-	sentry_tracing.c
-	sentry_tracing.h
 	)
 endif()

